Backend/src/main/java/com/project/omega/service/implmentations/OrderServiceImpl.java

```diff
@@ -48,24 +48,14 @@ public Order updateOrder(Order order) {
     }
 
 
+    public Order updateOrderById (Long id, Order order) throws NoRecordsFoundException {
+    if (!orderRepository.existsById(id)){
+        throw new NoRecordsFoundException("order" + id + "not Found");

```

Comment:
There's a messages_en.properties file containing all exception and other necessary messages. Refer to other service implementation files to see how to use them. You can also add your own messages in the messages_en.properties files if needed.

Reply:
added new message property and call for message in service